glutton/conntrack.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"github.com/MohammadBilalArif/glutton"
+	"github.com/hectane/go-nonblockingchan"
+	"log"
+	"os/exec"
+	"regexp"
+	"strconv"
+)
+
+const regexpression = `\[(\d+\.\d+)(?:\s+)?\]\s+\[\w+]\s+\w+\s+\w+\s\w+\s+.+?src=(\d+\.\d+\.\d+\.\d+)\s+dst=(\d+\.\d+\.\d+\.\d+)\s+sport=(\d+)\s+dport=(\d+)\s+`
+
+const connBufferSize int = 30000000
+
+func Monitor_Connections(proto string, channel *nbc.NonBlockingChan) {
```

Comment:
Yup, I will change this.  What about local variable and parameters. I have searched about naming conventions of Go Lang. I could follow [https://talks.golang.org/2014/names.slide#7](url) ?